user-manual/import-export/upload-digital-object.rst

```diff
@@ -191,6 +193,186 @@ your :term:`digital object`.
+   is coloured grey if the user does not have access and the hyperlink is not 
+   shown.
+
+   .. image:: images/digital-object-no-master-access.*
+      :align: center
+      :width: 90%
+      :alt: digital object without access to master representation 
+
+
+   .. TIP::
+
+      Access to the "Master file", "Reference copy", and "Thumbnail copy" can 
+      be configured per user group. The "Access master", "Access reference", 
+      and/or "Access thumbnail" permission can be set to "Grant" or "Deny" for 
+      any user group, including unauthenticated ("anonymous") users. See: :ref:`edit-user-permissions`. 
+
+8. Access to the "Master file", "Reference copy", and "Thumbnail copy" may be
+   overridden by actionable PREMIS rights configured for individual archival 
+   descriptions and their digital object. See :ref:`rights-digital-object`. 
+   For example, if a PREMIS right is added to an archival description where 
+   the "Act" is "Disseminate" and the "Restriction" is "Disallow", then the 
+   hyperlinks in the "Digital object metadata" section are no longer active, 
+   all the document icons are coloured grey, and the reference copy of the 
+   digital object is no longer displayed on the archival description view 
+   page. Instead the customizable text of the "Disallow statement" is
+   displayed at the top of the page as well as in a "Permissions" field for 
+   each Access copy representation, e.g. "Access to this record is restricted 
+   because it contains personal or confidential information. Please contact 
+   the Reference Archivist for more information on accessing this record.". 
+
+   .. image:: images/digital-object-premis-disallow.*
+      :align: center
+      :width: 90%
+      :alt: digital object PREMIS rights set to disallow
+
+   .. NOTE::
+
+      If the "Restriction" were to be changed to "Conditional" and the default 
+      permission settings are enabled then the "Reference copy" and "Thumbnail 
+      copy" become available again as hyperlinks and these representations are 
+      once again displayed on the AtoM templates. The customizable text for 
+      the "Condition Statement" is displayed in a "Permissions" field for the 
+      "Master file" representation, e.g. "This record has not yet been 
+      reviewed for personal or confidential information. Please contact the 
+      Reference Archivist to request access and initiate an access review.".
+
+:ref:`Back to top <upload-digital-object>`
+
+.. _digital-object-preservation:
+
+Digital object metadata for preservation files
+==============================================
+
+If you or another user of a preservation system (e.g. Archivematica) has sent a DIP upload to an archival description :term:`slug` in AtoM then thumbnails for the digital objects in that Dissemenation Information Package (DIP) will be displayed in the archival description's digital object carousel.
+
```

Comment:
Does it make sense here to make "Archivematica" a hyperlink to the AM homepage - and possibly to add a "for more information, see ..." (and then an intersphinx link to the DIP upload docs in AM), or else a SEEALSO with a link?

Reply:
Sounds good. Done.

Reply:
Never used intersphinx link before, not sure how to test if it works? I did make sure that `upload-atom` is the legit anchor for this in am-docs. So assuming that `:ref:`DIP upload <archivematica:upload-atom:>` will work?

Reply:
So yeah, the markup is there, not causing an error, but the link is not showing on my machine. Presumably because it's not configured to know that there is an intersphinx link to "archivematica:"?

Reply:
hmmm, i might need to check with the devs. I don't remember setting anything up manually - I thought that intersphinx was installed in the Vagrant box already. However, I also recall that you've got your own custom local setup, so perhaps you'll need to install the intersphinx lib?